mantis-control-plane/mantis-control-plane-server/src/main/java/io/mantisrx/server/master/persistence/IMantisStorageProvider.java

```diff
@@ -134,4 +136,8 @@ void storeAndUpdateWorkers(final IMantisWorkerMetadata existingWorker, final IMa
     List<String> initActiveVmAttributeValuesList() throws IOException;
 
     void setActiveVmAttributeValuesList(final List<String> vmAttributesList) throws IOException;
+
+    TaskExecutorRegistration getTaskExecutorFor(TaskExecutorID taskExecutorID) throws IOException;
```

Comment:
Let's sync offline to see how to resolve the internal cass problem for these new data required.

Reply:
+1 thanks!